Tests/BSONTests/BSONObjectIDTests.swift

```diff
@@ -48,7 +41,7 @@ final class BSONObjectIDTests: BSONTestCase {
 
     func testFieldAccessors() throws {
         let oid = try BSONObjectID("FEEEEEEEFBBBBBBBBBFAAAAA")
-        expect(oid.timestamp).to(equal(0xFEEE_EEEE))
+        expect(oid.timestamp).to(equal(Date(timeIntervalSince1970: 0xFEEE_EEEE)))
```

Comment:
Wasn't sure what to do here, since FEEEEEEE isn't really meant to represent a meaningful date and this test isn't directly meant to test that the timestamp is set correctly. Should I just delete this line? Leave it the way I did it? or more closely mimic the way `testTimestampCreation` checks the timestamp and ignore seconds?

Reply:
I think these tests are probably sufficient.